sdk/python/tests/unit/permissions/auth/server/conftest.py

```diff
@@ -1,15 +1,74 @@
 import os
 import tempfile
 from textwrap import dedent
+from typing import Optional, Union
 
 import pytest
 import yaml
 
-from feast import FeatureStore
+from feast import (
+    Entity,
+    FeatureStore,
+    FeatureView,
+    OnDemandFeatureView,
+    StreamFeatureView,
+)
+from feast.permissions.action import AuthzedAction
+from feast.permissions.permission import Permission
+from feast.permissions.policy import RoleBasedPolicy
 from tests.unit.permissions.auth.server.mock_utils import PROJECT_NAME
 from tests.utils.cli_repo_creator import CliRunner
 from tests.utils.http_server import free_port  # noqa: E402
 
+list_permissions_perm = Permission(
+    name="list_permissions_perm",
+    types=Permission,
+    with_subclasses=False,
```

Comment:
probably we don't need `with_subclasses` as there are no subclasses of `Permission`

Reply:
What exactly do you mean, to remove the line 26 entirely?